test/append-body.test.js

```diff
@@ -1,788 +0,0 @@
-'use strict'
-const test = require('tap').test
-const FormData = require('form-data')
-const Fastify = require('fastify')
-const multipart = require('..')
-const http = require('http')
-const path = require('path')
-const fs = require('fs')
-const pump = require('pump')
-
-const filePath = path.join(__dirname, '../README.md')
-
-test('addToBody option', t => {
```

Comment:
the `addToBody` option is needed to support automatic validation of the body itself if a user would like to do so. If we remove this functionality, we should add some equivalent replacement.

Reply:
JSON Schema was added.